### PR TITLE
chore(tests): simplify node tests

### PR DESCRIPTION
**Pull request by daniel-sanche**
- **removed extra log statements**, which were sometimes interpreted as library outputs and breaking tests 
   - https://github.com/googleapis/nodejs-logging/pull/1148
- **simplified how arguments are passed to snippets**. Now, each snippet receives a single "args" argument, which is a dictionary containing the arguments sent as part of the test
- **removed node-specific tests**. Node had two tests that didn't apply to other languages. (`requestlog`, `stdoutlog`). They seem to be testing code paths that should already be in the unit tests, and the way they were implemented complicates the environment test codebase. I removed them for now so we can grow the test codebase in a more consistent way for all languages

### CODE REVIEW

File: deployable/nodejs/app.js [hunk 1]
```diff
@@ -121,12 +121,6 @@ function triggerTest(message) {
...
 
-  console.log('Fn invoked with attributes, if any: ');
-  console.log(message.attributes);
-
-  if (message.attributes) {
-    tests[testName](message.attributes['log_name'], message.attributes['log_text']);
-  } else {
-    tests[testName]();
-  }
+  args = message.attributes ? message.attributes : {};
```
Comment (minherz) [inline deployable/nodejs/app.js:124]: i am not familiar with the logic here. but the fix may result in submitting more attributes than just the following two: `"log_name"` and `"log_text"`. is it the intended behavior?

File: tests/common/common.py [hunk 2]
```diff
@@ -173,25 +170,7 @@ def test_monitored_resource(self):
...
-        log_list = self.trigger_and_retrieve(log_text, "requestlog")
-        # Note: 2 logs are spawned, only one containing http_request prop.
-        log_entry = log_list[-1]
-        if log_entry.http_request is None:
-          log_entry = log_list[-2]
-        found_request = log_entry.http_request
-        if hasattr(self, 'request_props'):
-            for prop in self.request_props:
-                self.assertTrue(found_request[prop],
-                f'{prop} is not set')
```
Comment (minherz) [inline tests/common/common.py:189]: do we stop testing logging request metadata?

File: deployable/nodejs/app.js (hunk 1 above)
Comment (daniel-sanche) [reply deployable/nodejs/app.js:124]: Yes, that's the intention. We want to be able to trigger test functions with arbitrary arguments (I added severity here already, and Python makes use of others)

File: tests/common/common.py (hunk 2 above)
Comment (daniel-sanche) [reply tests/common/common.py:189]: I want to rethink how this is done. The goal of the environment tests is to ensure consistency across languages/environments, but this one is currently node-only. We should try to re-write it in a way that tests the same thing in all languages.

Note that for the node library specifically, request metadata is already better covered in [the system tests](https://github.com/googleapis/nodejs-logging/blob/864f18848c87a22c335b48e60a1d4505d3c9bd94/system-test/logging.ts#L620), so this one was a bit redundant

**Review by minherz — APPROVED** [review]

**Pull request merged**